src/components/calcite-label/calcite-label.tsx

```diff
@@ -28,7 +28,10 @@ export class CalciteLabel {
   /** specify the text alignment of the label */
   @Prop({ reflect: true }) alignment: Alignment = "start";
 
-  /** specify the status of the label and any child input / input messages */
+  /**
+   * specify the status of the label and any child input / input messages
+   * @deprecated status is no longer supported, set status directly in child input / input messages
```

Comment:
Pinging @benelan for editorial review. :)

Reply:
Can you please change the `@deprecated` message to `set directly on child element instead` to match the cadence of our other deprecation notices.